### PR TITLE
47 create a overview of databases and list all created mappings

### PR DESCRIPTION
**Pull request by MubaiHua**

### CODE REVIEW

File: backend/excelinterface/views.py [hunk 1]
```diff
@@ -372,6 +372,21 @@ def create(self, request, *args, **kwargs):
         except Exception as e:
             return Response({'message': 'Fail to create new database', 'error': e}, status=400)
 
+    @action(detail=False, methods=['post'])
```
Comment (samxu01) [inline backend/excelinterface/views.py:375]: good code

File: frontend/src/Components/Overview.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from 'react';
```
Comment (samxu01) [inline frontend/src/Components/Overview.jsx:1]: good structure

Comment (MubaiHua) [reply frontend/src/Components/Overview.jsx:1]: please approve

**Review by samxu01 — APPROVED** [review]
LGTM

**Pull request merged**